gui/src/components/alerts/alert-details.vue

```diff
@@ -40,10 +40,10 @@
       </div>
       <div class="cortx-text-md cortx-text-bold">
         <div>
-          <label>{{ $t("alerts.cluster") }} {{ alertExtendedInfo.cluster_id }}</label>
-          <label>&nbsp;| {{ $t("alerts.site") }} {{ alertExtendedInfo.site_id }}</label>
-          <label>&nbsp;| {{ $t("alerts.rack") }} {{ alertExtendedInfo.rack_id }}</label>
-          <label>&nbsp;| {{ $t("alerts.host_name") }}: {{ alert.hostname}}</label>
+          <label v-if="alertExtendedInfo.cluster_id">{{ `${ $t("alerts.cluster") } ${alertExtendedInfo.cluster_id} | ` }}</label>
```

Comment:
why we are removin nbsp?
please atatch a screenshot

Reply:
I have refactored the spacings using template literals. There will no change in the way it looks. I have already attached the screenshots (3rd and 4th).